### PR TITLE
Fix picker narrator and fix tabbing out of picker

### PR DESCRIPTION
**Pull request by MADLinsen**
- Fix picker narrator so it now actually narrates (tested in chrome using NVDA, still need to test narrator in edge/ie using windows narrator)
- Added setsize and posinset for narrator
- Fixed tabbing out of picker. This was disabled for some reason (expandableNode line 44 now removed).

### CODE REVIEW

File: src/components/treeView/leafNode.tsx [hunk 1]
```diff
@@ -45,13 +45,21 @@ export class LeafNode extends React.Component<LeafNodeProps, {}> {
 		return this.props.treeViewId + this.props.id;
 	}
 
+	private setsize() {
```
Comment (mttwc) [inline src/components/treeView/leafNode.tsx]: We can remove these methods and call the props directly from render

**Review by mttwc — APPROVED** [review]
Nice!
